ui/views/components/head.ejs

```diff
@@ -0,0 +1,72 @@
+<meta charset="utf-8" />
+<meta http-equiv="X-XSS-Protection" content="1;mode=block" />
+<meta http-equiv="X-Content-Type-Options" content="nosniff" />
+<meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
+<meta http-equiv="Referrer-Policy" content="strict-origin-when-cross-origin" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<meta name="theme-color" content="#235DF4" />
+<meta name="description" content="" />
+
+<title>DanaSwap App</title>
+
+<link rel="icon" href="./assets/images/favicon.ico" />
+
+<link rel="stylesheet" id="theme-css" href="./assets/styles/night.css" />
+
+<link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+/>
+
+<script>
```

Comment:
This need to be in a separate file so our CSP can deny `unsafe-inline` for JavaScript for the security of our users.

Reply:
CSP will deny JS generated by PS :disappointed:

Reply:
That can't be right. We should talk about what's going on here.